spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -983,8 +983,7 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
-  // TODO: enable this when we add md5 function to Comet
-  ignore("md5") {
+  test("md5") {
```

Comment:
Hmm, I remember we explicitly disable it because DataFusion `crypto_expressions` feature includes `blake3` which cannot be built on Mac platform.

Reply:
I don't see you add `crypto_expressions` to `Cargo.toml`, how does it work?

Reply:
> hmm, I think [`crypto_expressions`](https://github.com/apache/datafusion/blob/main/datafusion/core/Cargo.toml#L52) is enabled as default features in DataFusion?

Yes, but we don't use default features:

```
default-features = false
```

Reply:
> > I remember we explicitly disable it because DataFusion crypto_expressions feature includes blake3 which cannot be built on Mac platform.
> 
> Do you have any issues to track this one? I think I can build DataFusion by default on Apple Silicon.

Not sure if the crate is updated to fix that. We encountered the issue and disabled `crypto_expressions` one year ago (internally, before we open sourced Comet).

Maybe it is okay now. But I'm wondering as we don't add back `crypto_expressions` feature, is `md5` function working? I think it is guarded by this feature in DataFusion.